### ISSUE TITLE
Issue with Module v4l2loopback 

### ISSUE THREAD

**Issue by fbuffa2003** [linux-surface/linux-surface#973]
On this section https://github.com/linux-surface/linux-surface/wiki/Camera-Support#gstreamer-loopback-device-for-other-applications   I'm getting this error

 sudo modprobe v4l2loopback video_nr=42 card_label="virtualcam" exclusive_caps=1
modprobe: FATAL: Module v4l2loopback not found in directory /lib/modules/5.19.16-1.surface.fc36.x86_64

Any ideas?

### Environment

- Hardware model:  Surface Book 2
- Kernel version: 5.19.16
- Distribution: Fedora 36

**Comment by StollD** [comment]
v4l2loopback is an external module, it is not part of our kernel builds. You need to install it seperately. I think for fedora, it is packaged in rpmfusion.

**Comment by fbuffa2003** [comment]
Thanks for the quick reply and the support to the project. 

I just enable the repositories and seams to be installed. But still return a error 

https://docs.fedoraproject.org/en-US/quick-docs/setup_rpmfusion/

![image](https://user-images.githubusercontent.com/26152219/198872088-cfc94cb4-ab34-42e7-a705-cf81046dc32c.png)

**Comment by NP-chaonay** [comment]
> Thanks for the quick reply and the support to the project.
> 
> I just enable the repositories and seams to be installed. But still return a error
> 
> https://docs.fedoraproject.org/en-US/quick-docs/setup_rpmfusion/
> 
> ![image](https://user-images.githubusercontent.com/26152219/198872088-cfc94cb4-ab34-42e7-a705-cf81046dc32c.png)

maybe could you try to reinstall kernel package

**Comment by StollD** [comment]
Did you install `kernel-surface-devel`? You need that for external modules. Make sure that `v4l2loopback-kmod` is installed too.

After installing that try running `sudo akmods` and see what it outputs.

**Comment by fbuffa2003** [comment]
`kernel-surface-devel` was missing in my installation. 

v4l2loopback-kmod installed

result `sudo akmods`:

![image](https://user-images.githubusercontent.com/26152219/198874126-d166ba35-c0f4-49f1-a856-feb593f2d792.png)

Now all working! Thanks!

**Comment by fbuffa2003** [comment]
Sorry to reopen. now is working the command but the second is returning this error:
![image](https://user-images.githubusercontent.com/26152219/198874482-5bd4d7d7-f0ba-47aa-9bf7-0f8919f529d8.png)
I checked the devices and sems to be number 4. Thats correct?
![image](https://user-images.githubusercontent.com/26152219/198874523-94089087-ab75-4499-b4d6-62bd94c69017.png)

**Comment by StollD** [comment]
You need to use a number that doesnt already exist. Thats why the wiki entry uses 42.

**Comment by fbuffa2003** [comment]
That was my first try and same error show up:

![image](https://user-images.githubusercontent.com/26152219/198875315-5ceb557b-e973-44d3-b5f1-3f7dd3bfdd61.png)

**Comment by StollD** [comment]
Wait, you already have a virtual device (the OBS Virtual Camera). For some reason, the RPMFusion package automatically loads it. You can't load a module twice, so you either need to use that device (`/dev/video14`) or unload it before running the command from the wiki (`sudo modprobe -r v4l2loopback`)

**Comment by fbuffa2003** [comment]
Yes correct. So, I removed the module with `sudo modprobe -r v4l2loopback` and then running the command I see this:
![image](https://user-images.githubusercontent.com/26152219/198876048-081e87a7-ed25-4506-9497-ce2e6d8f5a0b.png)
The light of the camera is for few seconds on and then off. From the browser Brave no camera was detected.

**Comment by StollD** [comment]
You unloaded it but did you reload it with the params from the wiki?

**Comment by fbuffa2003** [comment]
Yes same parameters. Copy and paste

**Comment by fbuffa2003** [comment]
Here is the entire commands

![image](https://user-images.githubusercontent.com/26152219/198876443-a9ee7457-4812-4026-89b0-31193e4a71b4.png)

**Comment by fbuffa2003** [comment]
It helps a little bit. Camera was on for a second and then light off but got stuck in this line. At least the browser is detecting the camera but is black.

![image](https://user-images.githubusercontent.com/26152219/198876973-4b6a3d59-eb0e-4d55-8432-34606557cff9.png)

**Comment by catsdgs** [comment]
I am having this exact same issue with my Surface Pro 4.

**Comment by farline99** [comment]
Sometimes this problem also happens to me. It was always a problem, when I manually was starting pipeline after one timeline before (after I used timeline in this reboot), only reboot helped. After that I created systemctl script and start it when I use camera, stop when not, it is working without problems. Surface Pro 6, Arch Linux.
